specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -1450,20 +2063,43 @@
       "required": true,
       "type": "string"
     },
+    "resourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "description": "Azure Resource Group Name.",
+      "required": true,
+      "type": "string"
+    },
     "budgetNameParameter": {
-      "name": "name",
+      "name": "budgetName",
       "in": "path",
-      "description": "Budget name.",
+      "description": "Budget Name.",
       "required": true,
-      "type": "string",
-      "x-ms-parameter-location": "method"
```

Comment:
How I wish this linter rule is implemented https://github.com/Azure/azure-openapi-validator/issues/84

Reply:
Thanks @amarzavery .. we will make this fix in PR https://github.com/Azure/azure-rest-api-specs/pull/2386